storageFormats/eventMetadata.cc

```diff
@@ -316,6 +316,22 @@ eventMetadata* rpwa::eventMetadata::merge(const vector<const eventMetadata*>& in
 	if(mergeDiffMeta) {
 		mergee->setBinningMap(mergedBinningMap);
 	}
+
+	for (const auto& input : inputData) {
```

Comment:
Thanks a lot for your comment.
I fully agree that one should be cautious with using `auto`, but in the context of for loops I thinks it is very useful, because you usually spare lots of characters, which makes the code much more readable.